R/generate_simulated_data.R

```diff
@@ -381,7 +392,7 @@ generate_simulated_data <- function(r_in_weeks = # nolint
     log_r_site_aux <- construct_aux_rt(log_r_state_week,
       state_deviation_ar_coeff = phi_rt,
```

Comment:
I think this would be slightly more readable if you specified the function arguments and passed them in e.g.
`log_r_site_aux <- construct_aux_rt(log_r_state_week = log_r_state_week) ` etc (so the same as you have for the second argument, but for all of them.

Reply:
I know this isn't done for everything else here, it will be once #29 is merged into the main codebase and into this....